src/uu/mv/src/mv.rs

```diff
@@ -765,9 +719,9 @@ fn rename(
             if is_empty_dir(to) {
                 fs::remove_dir(to)?;
             } else {
-                return Err(io::Error::other(get_message(
-                    "mv-error-directory-not-empty",
-                )));
+                return Err(io::Error::other(
+                    translate!("mv-error-directory-not-empty", "directory" => to.quote()),
```

Comment:
Is this intentional?
```suggestion
                    translate!("mv-error-directory-not-empty"),
```